pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation-local.adoc

```diff
@@ -1,105 +0,0 @@
-[[apim-kubernetes-operator-installation-local]]
```

Comment:
Why has this page been deleted?

Reply:
Every references to the installation in a local cluster using dev tools like k3d should not be referenced in the official documentation. It makes the usage of our product and the documentation confusing. We agreed in the team that everything related to dev tools should be only documented on GitHub in the project documentation.